src/auth/entities/user.entity.ts

```
@@ -39,7 +39,9 @@ export class User {
     example: 'hashed_password',
   })
   @Column()
-  password: string;
+
+  password: string
+
```

ESLint

[error] Replace `⏎··password:·string⏎` with `··password:·string;`

(prettier/prettier)